controls/stig_rhel9.yml

```diff
@@ -958,6 +958,7 @@ controls:
             - medium
         title: RHEL 9 cron configuration directories must have a mode of 0700 or less permissive.
         rules:
+            - package_cron_installed
```

Comment:
The STIG only calls out the permissions not the package install.

What problem are you trying to solve?

Reply:
I'm trying to solve the problem that the rules checking these files fail because the files don't exist because the package that provides these files isn't installed.